doom-modeline-segments.el

```diff
@@ -472,12 +472,130 @@ mouse-1: Previous buffer\nmouse-3: Next buffer"
 ;; Indentation
 ;;
 
+;; Indentation retrieving variables matched to major modes. It is based upon
+;; `editorconfig-indentation-alist' (https://github.com/editorconfig/editorconfig-emacs) but is used
+;; to read indentation levels instead of setting them. When multiple variables are specified, they
+;; will be tried resolved in the given order.
+(setq doom-modeline--indent-alist
```

Comment:
I will make it into a
```el
(defcustom doom-modeline-indent-alist
  '((apache-mode apache-indent-level)
    (awk-mode c-basic-offset)
    ;; ...snip...
    (yaml-mode yaml-indent-offset))
  "Indentation retrieving variables matched to major modes used
  when `doom-modeline-indent-info' is non-nil. When multiple
  variables are specified, they will be tried resolved in the
  given order."
  :type '(alist :key-type symbol :value-type sexp)
  :group 'doom-modeline)
```

So that it's easier for users to extend.

Reply:
cool!